src/lib/Scenes/City/Components/CityTab.tsx

```diff
@@ -46,19 +41,19 @@ export class CityTab extends React.Component<Props> {
       case "Saved":
         return (
           <Box py={2}>
-            <StyledMessage textSize="3t">{`You haven’t saved any shows in ${cityName}. When you save shows, they will show up here.`}</StyledMessage>
+            <Message textSize="3t">{`You haven’t saved any shows in ${cityName}. When you save shows, they will show up here.`}</Message>
```

Comment:
So these are already centered?

Reply:
No the component actually doesn't support centered text but got the OK from Owen that it's not necessary.

Reply:
Okeydokey